require-config.js

```diff
@@ -1,21 +1,20 @@
 require.config( {
```

Comment:
If this file is only used in (the old) app.js, I guess we should get rid of entirely.